src/task_manager/graph.py

```
@@ -31,6 +29,17 @@
 ) -> Coroutine[Any, Any, dict]:
+    """
+    Creates an asynchronous function that retrieves recent user memories, formats them into a prompt, and invokes a language model with contextual information.
+    
+    The returned coroutine, when called, searches the memory store for recent relevant memories based on the user's latest messages, embeds these memories and the current timestamp into a system prompt, and calls the provided language model with this prompt and the conversation history.
+    
+    Args:
+        llm_with_tools: A runnable language model instance capable of tool use.
+    
+    Returns:
+        An asynchronous function that accepts the current state, configuration, and optional memory store, and returns a dictionary containing the model's response message.
+    """
```

Ruff (0.8.2)

Multi-line docstring summary should start at the first line

Remove whitespace after opening quotes

(D212)